src/featbin/wav-reverberate.cc

```diff
@@ -0,0 +1,250 @@
+// featbin/extract-segments.cc
```

Comment:
It would be difficult to run the unit test from the command line if it is in the same binary, so please add it to a new binary. (wav-reververate-test)